frontend/src/pages/Register.js

```diff
@@ -41,28 +43,36 @@ const Register = () => {
   if (registerStatus === LOADING_VALIDATE_INVITEID)
     return <LoadingIndicator text="Loading session..." />;
 
-  // TODO: better UI
-  if (registerStatus === VALIDATE_INVITEID_FAILED)
-    return <div>Wrong invite id</div>;
+  if (registerStatus === VALIDATE_INVITEID_FAILED) {
+    return (
+      <Layout cols={1}>
+        <Header text="Wrong invite id" />
+      </Layout>
+    );
+  }
 
-  // TODO: make text look better
   return (
     <div className={s.register}>
       <Layout cols={1}>
         <Header text="Finish your registration" isLoading={registerStatus === LOADING_REGISTER} />
-        <Text>
-          {`Hey, ${invitedUser.firstName} ${invitedUser.lastName} (${invitedUser.email})
-          You have been invited by ${inviterUser.firstName} ${inviterUser.lastName} (${inviterUser.email})`}
-        </Text>
-        {showNotification && (
-          <Notification
-            type="success"
-            text="Registration successful"
-          />
-        )}
+        <div align="center">
+          <Text
+            skin="disabled"
+            size="small"
+          >
+            {`Hey, ${invitedUser.firstName} ${invitedUser.lastName} (${invitedUser.email})
```

Comment:
This text is barely visible on my screen. You can use `<span style={{color: '...'}}>text</span>` inside the <Text> to get have any color. Also, can we have **bold** or another color to highlight firstName+lastName of both invitedUser and inviterUser? I think it would look awesome